index.html

```diff
@@ -69,33 +67,85 @@
     SHOULD join the mailing lists below and take part in the discussions.</p>
   </section>
   <section>
+      <h2>
+        Dependencies
+      </h2>
+      <p>
+        The terms <dfn><a href=
+        "https://html.spec.whatwg.org/multipage/semantics.html#the-link-element">link</a></dfn>, 
+        <dfn><a href="https://html.spec.whatwg.org/multipage/infrastructure.html#insert-an-element-into-a-document">inserted into a document</a></dfn>, 
+        <dfn><a href="https://html.spec.whatwg.org/multipage/infrastructure.html#resolve-a-url">resolve</a></dfn>, <dfn><a href=
+        "https://html.spec.whatwg.org/multipage/infrastructure.html#url">url</a></dfn>, <dfn><a href=
+        "https://html.spec.whatwg.org/multipage/semantics.html#attr-link-crossorigin">crossorigin</a></dfn>,
+        <dfn><a href="https://html.spec.whatwg.org/multipage/browsers.html#origin-2">origin</a></dfn>, <dfn><a href=
+        "https://html.spec.whatwg.org/multipage/parsing.html#delay-the-load-event">delay the load event</a></dfn>, <dfn><a href=
+        "https://html.spec.whatwg.org/multipage/webappapis.html#queue-a-task">queue a task</a></dfn> and <dfn><a href=
+        "https://html.spec.whatwg.org/multipage/semantics.html#concept-link-obtain">obtain the resource</a></dfn>  are defined in [[!HTML5]].
```

Comment:
You link to WHATWG HTML, but then claim to cite HTML5. That's not great. Please just cite [[!HTML]].